azure-devops/azext_devops/dev/common/artifacttool_updater.py

```diff
@@ -158,10 +158,18 @@ def _get_current_release(organization, override_version):
     # Distro returns empty strings on Windows currently, so don't even send
     distro_name = distro.id() or None
     distro_version = distro.version() or None
+    os_name = platform.system()
+    arch = platform.machine()
+
+    # For M1 macs, there is no version of artifact tool. However, the x86_64
+    # version can run under Rosetta, so we use that instead.
+    if os_name == "Darwin" and arch == "amd64":
```

Comment:
Hi @dalemyers I tested using your branch with a M1 processor but it didn't work, so I changed from amd64 to arm64 (M1 arch), rebuilt and azure can respond to M1 based requests. Would you consider adding arm64?

Reply:
Ah, that's likely a mistake on my part. I haven't yet managed to test this on an M1 mac, so wouldn't have seen this typo. I'll update with both to be sure. Thanks for the heads up!